vignettes/chk.Rmd

```diff
@@ -16,16 +16,57 @@ knitr::opts_chunk$set(
 
 ## Introduction
 
-`chk` is an R package for developers to check user-supplied function arguments.
+R functions and packages are great for sharing code,
+but it's hard to predict how end-users will implement them.
```

Comment:
```suggestion
R functions and packages are great for sharing code, but it's hard to predict how end-users will implement them.
```

@flor14 The changes you made are great. The only change I have is can you have each sentence on a single line like I have done in the code suggestion above. There is no need to have text span less than 80 characters in .Rmd files and it makes it easier to track changes if each sentence is it's own line.

Reply:
Solved in 05adee4